src/plugins/discover/public/application/components/sidebar/discover_field_search.test.tsx

```diff
@@ -145,33 +145,4 @@ describe('DiscoverFieldSearch', () => {
     popover = component.find(EuiPopover);
     expect(popover.prop('isOpen')).toBe(false);
   });
-
-  test('click outside popover should close popover', () => {
```

Comment:
@majagrubic looks like you added this test, wanted to comment on why it was removed

This tested functionality largely internal to EUI (that clicking elsewhere calls the closePopover callback). The change of `ownFocus`'s default value from false to true introduces **EuiFocusTrap** into the mix here, which causes this test to fail as the focus trap's document-level handlers aren't triggered by jest tests (see https://github.com/elastic/eui/pull/4228#issuecomment-723211025).

Because this tested internal functionality, and that it is now very difficult to test this situation from jest, I removed this test. Open to discussing other approaches if this test is important.